### PR TITLE
Devupdate v2

### PR DESCRIPTION
**Pull request by RaphaelBS-WCS**
- update h1 css
- update links
- update story responsive
- update section height
- modify alt in Nathalie's picture

### CODE REVIEW

**Review by Erdnaxela45 — DISMISSED** [review]
Fantasmagorique

File: src/View/Home/_story.html.twig [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
-                <img src="assets/images/story/photo_nathalie2.webp" class="img-fluid" alt="image of girls in pink purple background image">
-            </div>
-            <div class="col-12 col-sm-12 col-md-6 col-lg-6 ">
-                <p>Depuis l'âge de 8 ans, je perçois des sons ainsi que des visions. 
-                    A cette époque cela était difficile à appréhender ce phénomène. 
-                    Puis ne pouvant pas m'en défaire, j'ai appris à l'accepter et à le maîtriser. 
-                    C'est pourquoi, je vous propose de partager mon don et de lire à travers les cartes votre avenir.
-                    <a href="/history/index">En savoir+</a>
-                </p>
+    <div class="box_story">
```
Comment (LucyMerlier) [inline src/View/Home/_story.html.twig:2]: kebab-case please

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comment

File: src/View/Home/_story.html.twig (hunk 1 above)
Comment (RaphaelBS-WCS) [reply src/View/Home/_story.html.twig]: corrected

**Review by Erdnaxela45 — APPROVED** [review]

**Review by Vannou28 — APPROVED** [review]
tt'is good

**Review by LucyMerlier — APPROVED** [review]

**Pull request merged**